docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -7,7 +7,7 @@ import { alpha } from '@mui/material/styles';
 export default function AppFrameBanner() {
   return FEATURE_TOGGLE.enable_docsnav_banner ? (
     <Link
-      href={ROUTES.careers} // Fix me!
+      href={ROUTES.advancedComponentsWhatsNew}
```

Comment:
Probably simple enough:

```suggestion
      href="https://next.mui.com/x/whats-new/"
```